composer.lock

```diff
@@ -51,7 +52,7 @@
                 }
             ],
             "description": "Braintree PHP Client Library",
-            "time": "2017-08-25T19:38:09+00:00"
+            "time": "2017-08-25 19:38:09"
```

Comment:
Such changes are coming when old composer version used. Could you update your composer to latest version and run composer update?

Reply:
Also, it is worth updating `composer.json` at the framework level.
I'll do these changes on our side.